### ISSUE TITLE
Mix um Arithm STD and Geo STD?

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/OSPSuite.Core#2052]
@Yuri05 This is BS right?

![image](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/assets/1041237/9a207134-3b42-4ee1-a250-a7d1da02cdff)

**Comment by msevestre** [comment]
oh my bad
![image](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/assets/1041237/e166ec70-6dca-403b-981b-0ec4bff64498)


Can we set the default somehow? PKSim assumes geto std later on

**Comment by Yuri05** [comment]
Yep, STD intervals seem to be plotted for the **arithmetic** STD while it should be **geometric**.
